packages/docusaurus/lib/server/load/plugins.js

```diff
@@ -42,7 +42,7 @@ module.exports = async function loadPlugins({pluginConfigs = [], context}) {
       const content = await plugin.loadContent();
       const pluginContentPath = path.join(name, metadataFileName);
       const pluginContentDir = path.join(context.generatedFilesDir, name);
-      fs.ensureDirSync(pluginContentDir);
+      await fs.ensureDir(pluginContentDir);
```

Comment:
If we add an `await` here, there's no difference between sync or not - the order and duration is going to be the same.